README.md

```diff
@@ -371,6 +371,13 @@ script {
 //...
 ```
 
+# Known issues
+## Adding credentials to a new CloudBees CD configuration  
+[BEE-27725] When creating a new Cloudbees CD configuration, you cannot add a new credential using the "Add" button.
+### Workarounds
+The following workarounds may be used instead of the "Add" button to help you add a credential:
+- The "Add" button works for existing configurations. So, create your configuration without the credential and then edit the configuration.
+- Add your credential in any other place. 
```

Comment:
Maybe adding link to off Jenkins doc of how to create credentials would be better
https://www.jenkins.io/doc/book/using/using-credentials/#configuring-credentials

Reply:
@alexey-sinyuk, 
Does this work?
```suggestion
- Add your credentials as described in Jekins' [Configuring credentials](https://www.jenkins.io/doc/book/using/using-credentials/#configuring-credentials).
```